x-pack/test/security_solution_cypress/cypress/tasks/create_new_rule.ts

```diff
@@ -126,11 +127,15 @@ import { waitForAlerts } from './alerts';
 import { refreshPage } from './security_header';
 import { EMPTY_ALERT_TABLE } from '../screens/alerts';
 
+export const createAndEnableRuleOnly = () => {
+  cy.get(CREATE_AND_ENABLE_BTN).click({ force: true });
```

Comment:
It should be safe to omit `{ force: true }`.

Reply:
Good call. I'll update.